readme.md

```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Ivakhovd.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Ivakhovd.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="927" alt="image" src="https://github.com/user-attachments/assets/cdd11441-0d1b-41a0-9c61-cd20201ab539">